Marlin/Conditionals.h

```diff
@@ -203,13 +210,11 @@
   #endif
```

Comment:
Yep, and this can be in the block below too, so…
```cpp
  /**
   * LCD contrast for dogm-like LCD displays
   */
  #if ENABLED(DOGLCD)
    #define HAS_LCD_CONTRAST (DISABLED(U8GLIB_ST7920) && DISABLED(U8GLIB_SSD1306) && DISABLED(U8GLIB_SH1106))
    #if DISABLED(DEFAULT_LCD_CONTRAST)
      #define DEFAULT_LCD_CONTRAST 32
    #endif
  #endif
```